src/backend/utils/indexer.js

```diff
@@ -2,7 +2,7 @@ require('../lib/config');
 
 const { setIntervalAsync, clearIntervalAsync } = require('set-interval-async/dynamic');
 
-const { ELASTIC_MAX_RESULTS } = process.env;
+const { ELASTIC_MAX_RESULTS_PER_PAGE } = process.env;
```

Comment:
We need to set a default here so we don't blow up if the env is missing this.  I'd suggest:

```js
const { ELASTIC_MAX_RESULTS_PER_PAGE = 10 } = process.env;
```

Reply:
i think we decided on 5 yesterday, so I'll set it to 5?

Reply:
Correct, I'm wrong with 10.